src/app.ts

```diff
@@ -1,5 +1,19 @@
+#!/usr/bin/env node
+
+// first line is mandatory in JS, not sure about TS
+
+// try to run this with
+// node out/app.js -w 0:99 -d localhost:1234 -a db:pwd -b acc:container -k blob-key -t 1 -c crawl
+// node out/app.js -h
+
 import {ProcessingManager} from "./processing-manager";
 import {Worker} from "./worker";
+import {CLI} from "./cli";
+
+
+CLI.initCLI();
+CLI.logParms(); // will hide sensitive parameters, never log CLI.parameters directly!
```

Comment:
This will cause all kinds of problems. `Worker`s are exact copies of our application and will use `app.ts` as their entry point. Therefore they will run everything in there, including the `CLI` module. The `CLI` has to be initialized within `ProcessingManager.run()` or needs to be wrapped with an `if (cluster.isMater) { ... }` statement. I would strongly advise against using the latter though as it makes the code very confusing to read!